ff-outro.md

```diff
@@ -0,0 +1,121 @@
+
+(From [Understanding the GitHub flow](https://guides.github.com/introduction/flow/))
+
+When unresolvable disagreements break out, one can maintain the branch derivative as a permanent _fork_ to evolve the software project to a different trajectory. The practice of forking is why we have a diverse number of Linux distributions.
+
+![linux distros](https://specials-images.forbesimg.com/imageserve/5dc1a7ccca425400079c78c4/960x0.jpg?fit=scale)
+
+(By Eric Adams through [Forbes](https://www.forbes.com/sites/jasonevangelho/2019/11/05/my-search-for-the-perfect-linux-os-just-ended--with-an-unexpected-surprise/?sh=2798c7045cd6))
+
+### GitHub
+
+However, due to a barrier to converting from a centralized mindset to a decentralized one from its users, `git` was notoriously convoluted and hard to utilize. Toward the end of 2007, a group of software developers who enjoyed `git` but envisioned better ways to use it started building what's known as GitHub today, making the usage pleasant for more. They have been building GitHub using `git` and GitHub itself, a practice known as [dogfooding](https://en.wikipedia.org/wiki/Eating_your_own_dog_food).
+
+The company started as profitable from day one, refrained from taking external funding until [July 2012](https://www.reuters.com/article/github-fundraising/github-raises-100-mln-from-andreessen-horowitz-idUSL2E8I9AV320120709), and stayed as a flat organization with 234+ employees until [17 July 2014](https://en.wikipedia.org/wiki/Timeline_of_GitHub).
+
+Perhaps the root of their success is a culture that emphasizes [people's happiness instead of money](https://tom.preston-werner.com/2010/10/18/optimize-for-happiness.html). The culture reflects in their ability to [develop quality products](https://github.blog/changelog/) that embrace and encourage the best practices from the open-source community and benefit even proprietary developments. Not surprisingly, they are also a large contributor to various open-source software projects, with an adopted [Open Source (Almost) Everything](https://tom.preston-werner.com/2011/11/22/open-source-everything.html) mindset.
+
+Such an interpretation of open-source practices allows the company to attract top talents worldwide to collaborate on a service platform enjoyed by their users. By 2010 (a little over three years since inception), GitHub has claimed to be the largest host of source code in the world until today.
+
+## EQ Works - our adaptation of open-source
+
+A key idea behind the open-source movement is to leverage their learnings to make information and curation more transparent and efficient. This social movement coincides with the phenomenon of the internet-driven information explosion and the increasing realization of how closed platforms could [unethically profit from manipulating the information flow and shape](https://en.wikipedia.org/wiki/Facebook%E2%80%93Cambridge_Analytica_data_scandal).
```

Comment:
suggestion for "This social movement coincides with the phenomenon of the internet-driven information explosion and the increasing realization of how closed platforms could [unethically profit from manipulating the information flow and shape]": 

"This social movement coincides with increased access to information online and growing realization of how closed platforms could unethically profit from manipulation the information flow and shape."